### PR TITLE
BC-2807 - validate beta task

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

https://ticketsystem.dbildungscloud.de/browse/BC-2807

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/pages/TaskCard.page.vue [hunk 1]
```diff
@@ -328,6 +330,13 @@ export default defineComponent({
...
 
 		const save = async () => {
+			if (form.value) {
+				const valid = form.value.validate();
+				if (!valid) {
+					return;
+				}
+			}
+
 			if (hasErrors.value) {
```
[the remark this answers is not in the document]
Comment (odalys-dataport) [reply src/pages/TaskCard.page.vue:392]: Will be adjusted in [BC-2509](https://ticketsystem.dbildungscloud.de/browse/BC-3509)

File: src/components/card-elements/TitleCardElement.vue [hunk 2]
```diff
@@ -46,18 +63,28 @@ export default defineComponent({
 
 		const handleInput = () => emit("input", title.value);
 
+		const TITLE_MAX_LENGTH = 400;
+		const rules = {
+			required: (value: string) =>
```
[the remark this answers is not in the document]
Comment (odalys-dataport) [reply src/components/card-elements/TitleCardElement.vue:68]: Will be adjusted in [BC-2509](https://ticketsystem.dbildungscloud.de/browse/BC-3509)

File: src/components/card-elements/TitleCardElement.vue [hunk 3]
```diff
@@ -46,18 +63,28 @@ export default defineComponent({
 
 		const handleInput = () => emit("input", title.value);
 
+		const TITLE_MAX_LENGTH = 400;
```
[the remark this answers is not in the document]
Comment (christian-darsow) [reply src/components/card-elements/TitleCardElement.vue:66]: As we agreed on just use it for now only here in beta  task card context, we keep it like this

File: src/components/molecules/Alert.vue [hunk 4]
```diff
@@ -19,7 +19,13 @@
 			border="left"
 			@input="closeNotification"
 		>
-			<div class="alert_text mr-2">
+			<div v-if="messages" class="alert_text mr-2">
```
Comment (christian-darsow) [inline src/components/molecules/Alert.vue:22]: I would suggest to add one more test for this new possibilty to use the component

**Pull request merged**